client/my-sites/plugins/not-available-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NotAvailableError( { translate } ) {
+	return (
+			<Main>
+				<DocumentHead title={ translate( 'Plugins' ) } />
+				<SidebarNavigation />
+				<EmptyContent
+					title={ translate( 'Not Available' ) }
+					line= { translate( 'The page you requested could not be found' ) }
```

Comment:
@rickybanister @michelleweber thanks for chiming in.

As @jorgefilipecosta explained, that case is **not** about the Manage JP module being disabled. It's for when the user does not have the right capabilities to manage plugins (that usually happens when they are not an administrator - any of the default roles that is not a super admin or admin can't install/(de)activate/delete plugins). I just wanted to confirm we're on the same page 😉 

Feel free to ping again if you need any more information - these situations are sometimes not the simplest thing to wrap one's head around.